docs/examples/banneroverlay/main.js

```diff
@@ -0,0 +1,43 @@
+// @flow strict
+import { type Node as ReactNode, useState } from 'react';
+import { BannerOverlay, Button, FixedZIndex, Image, Link, Text } from 'gestalt';
+
+export default function Example(): ReactNode {
+  const [showComponent, setShowComponent] = useState(true);
+
+  return !showComponent ? (
+    <Button
+      onClick={() => {
+        setShowComponent(true);
+      }}
+      text="Show BannerOverlay"
+    />
+  ) : (
+    <BannerOverlay
+      zIndex={new FixedZIndex(100)}
+      offset={{ top: 130, bottom: 24 }}
```

Comment:
I'm not sure if we should expose offset. In Pinboard, we have a "ToastManager" for positioning. Is it like toast at the same location every time across all surfaces?

I think if we let people set this, they'll start putting it in funky places

Reply:
I was initially against this too, but after considering the options with Alberto this seemed like the best tradeoff (the final prop version was suggested by him heheh).
This could be also implemented in Pinboard with a gestaltExtension redefining the API and implementing something like a BannerOverlayManager as well.